sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -812,7 +812,7 @@ private async Task RunReceiveTaskAsync(CancellationToken cancellationToken)
                         // hold onto all the tasks that we are starting so that when cancellation is requested,
                         // we can await them to make sure we surface any unexpected exceptions, i.e. exceptions
                         // other than TaskCanceledExceptions
-                        var linkedCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
+                        var linkedCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, CancellationToken.None);
```

Comment:
Mind porting this over to the other PR so we can just run the test suite there?

Reply:
I'd like to sneak this into the 7.6.0 release.

Reply:
Sure can do. Will close this one and cherry pick over to #26904